src/window.rs

```diff
@@ -46,7 +46,7 @@ impl fmt::Debug for WindowId {
 }
 
 /// Attributes used when creating a window.
-#[derive(Debug, Clone, PartialEq)]
```

Comment:
I think there's no real value in `PartialEq` on that, and maintaining it is a hustle.

Reply:
I mean it's a giant opaque object, you'd generally need to just `Clone` or check individual fields, comparing the entire thing doesn't really make any sense.